tests/api_tests/users/test_views.py

```diff
@@ -191,3 +191,145 @@ def test_get_projects_logged_in_as_different_user(self):
+        url = "/{}users/{}/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+    def test_get_200_path_users_me_user_logged_in(self):
+        url = "/{}users/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+    def test_path_users_me_nodes_user_logged_in(self):
+        url = "/{}users/me/nodes/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+        node_json = res.json['data']
+
+        ids = {each['id'] for each in res.json['data']}
+        assert_in(self.public_project_user_one._id, ids)
+        assert_in(self.private_project_user_one._id, ids)
+        assert_not_in(self.public_project_user_two._id, ids)
+        assert_not_in(self.private_project_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.deleted_folder._id, ids)
+        assert_not_in(self.deleted_project_user_one._id, ids)
+
+    def test_path_users_user_id_nodes_user_logged_in(self):
+        url = "/{}users/{}/nodes/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+
+        node_json = res.json['data']
+
+        ids = [each['id'] for each in node_json]
+        assert_in(self.public_project_user_one._id, ids)
+        assert_in(self.private_project_user_one._id, ids)
+        assert_not_in(self.public_project_user_two._id, ids)
+        assert_not_in(self.private_project_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.deleted_folder._id, ids)
+        assert_not_in(self.deleted_project_user_one._id, ids)
+
+    def test_get_404_path_users_user_id_me_user_logged_in(self):
+        url = "/{}users/{}/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_404_path_users_user_id_me_user_not_logged_in(self):
+        url = "/{}users/{}/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_two, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_404_path_users_user_id_nodes_me_user_logged_in(self):
+        url = "/{}users/{}/nodes/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_one, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_404_path_users_user_id_nodes_me_user_not_logged_in(self):
+        url = "/{}users/{}/nodes/me/".format(API_BASE, self.user_one._id)
+        res = self.app.get(url, auth=self.auth_two, expect_errors=True)
+        assert_equal(res.status_code, 404)
+
+    def test_get_404_path_nodes_user_id_user_logged_in(self):
```

Comment:
While these certainly don't hurt anything other than our test execution time, I'm not sure these tests are necessary (gasp). There are an infinite ways we can generate 404 responses--unless the test is providing useful verification of intended behavior, we can probably omit them.

That said, I think it's fine to leave these for now.